frontend/common/src/components/UserProfile/SkillAccordion/SkillAccordion.tsx

```diff
@@ -199,13 +199,14 @@ const SkillAccordion: React.FunctionComponent<SkillAccordionProps> = ({
         <p>
           {intl.formatMessage(
             {
-              defaultMessage: "<primary>{role}</primary> at {division}",
-              description: "Role at Team, Group or Division",
+              defaultMessage: "<primary>{role}</primary> at {organization}",
```

Comment:
This will break the French string equivalent in fr.json, are we waiting to have this translated in another round later on?

Reply:
Can't we just change the `{division}` value to `{organization}` in the fr.json file? And fix the description?

Reply:
Oui, but you will need to regenerate (extract) the string if you change the value of _defaultMessage_ or _description_.